src/compiler/checker.ts

```diff
@@ -11309,7 +11311,12 @@ namespace ts {
                         }
                     }
                 }
-
+                // Do not use hasDynamicName here, because that returns false for well known symbols.
+                // We want to perform checkComputedPropertyName for all computed properties, including
+                // well known symbols.
+                if (node.name.kind === SyntaxKind.ComputedPropertyName) {

```

Comment:
this doesn't do the `node.name && ...` check that `checkFunctionLikeDeclaration` does. Is this because `node.name` is guaranteed to be defined for accessors?